charts/spark-operator-chart/templates/deployment.yaml

```diff
@@ -47,6 +47,8 @@ spec:
       - name: {{ .Chart.Name }}
         image: {{ .Values.image.repository }}:{{ default .Chart.AppVersion .Values.image.tag }}
         imagePullPolicy: {{ .Values.image.pullPolicy }}
+        envFrom:
+          {{- toYaml .Values.envFrom | nindent 12 }}
```

Comment:
Should 12 be consistent with 10 below?

Reply:
Nice catch! Updated. We have it running with 12 but 10 should also work and is more consistent. Thanks!